app/src/main/java/org/jellyfin/androidtv/ui/preference/custom/ColorPickerDialogFragment.kt

```diff
@@ -0,0 +1,185 @@
+package org.jellyfin.androidtv.ui.preference.custom
+
+import android.content.res.ColorStateList
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Checkable
+import android.widget.RadioButton
+import android.widget.TextView
+import androidx.appcompat.view.ContextThemeWrapper
+import androidx.core.content.ContextCompat
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.view.isVisible
+import androidx.leanback.preference.LeanbackPreferenceDialogFragmentCompat
+import androidx.leanback.widget.VerticalGridView
+import androidx.preference.ListPreference
+import androidx.recyclerview.widget.RecyclerView
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.databinding.PreferenceColorListBinding
+
+class ColorPickerDialogFragment : LeanbackPreferenceDialogFragmentCompat() {
+	companion object {
+		fun newInstance(key: String) = ColorPickerDialogFragment().apply {
+			arguments = Bundle().apply {
+				putString(ARG_KEY, key)
+			}
+		}
+	}
+
+	enum class LighterColors(val color: Long) {
+		// Use named parameter so detekt knows these aren't magic values
+		COLOR_WHITE(color = 0xFFFFFFFF),
+		COLOR_YELLOW(color = 0xFFEEDC00)
+	}
```

Comment:
I don't really see a need to use enums for the colors. especially if it's just two.

Reply:
Just suppress them, if they're clearly labelled they are false-positives